bridge/client/app/_components/ktb-keptn-services-list/ktb-keptn-services-list.component.html

```diff
@@ -1,4 +1,4 @@
-<div fxFlexFill fxLayout="column" class="container">
+<div fxFlexFill fxLayout="column" class="container" *ngIf="projectName$ | async as projectName">
   <div fxLayout="column" fxFlex="0 0 60%" class="overflow-y-scroll">
     <dt-loading-spinner *ngIf="!uniformRegistrations.data"></dt-loading-spinner>
```

Comment:
It shows "No integrations available" when loading data and not the loading spinner - which is confusing.

Reply:
is fixed now